week2/InterfaceExample/MyInterface.java

```diff
@@ -0,0 +1,44 @@
+public interface MyInterface {
+     public final int speed=90;
+    default void speedLimit()
+    {
+        System.out.println("Speed limit is "+speed+" KMPH");
+    }
+    public abstract void company();
+}
+class Bike1 implements MyInterface
+{
+    @Override
+    public void company()
+    {
+        System.out.println("From Royal Enfield");
+    }
+}
+
+class Bike2 implements MyInterface
+{
+    @Override
+    public void company()
+    {
+        System.out.println("From KTM");
+    }
+}
+class interfaceDemo implements MyInterface
+{
+    @Override
+    public void company()
+    {
+        System.out.println("Manufacturing companies");
+    }
+
+    public static void main(String[] args) {
+        interfaceDemo id = new interfaceDemo();
+        id.company();
+        Bike1 b1 = new Bike1();
+        b1.company();
+        b1.speedLimit();

```

Comment:
Use interface as reference for all Bike implemention class, pls follow the requirement as given restrict the speed limit of a bike to 90 KMPH

Reply:
fixed